src/angularComponent/ngcOmniboxSuggestionItemDirective.js

```diff
@@ -4,13 +4,19 @@ export default function ngcOmniboxSuggestionsDirective() {
   return {
     restrict: 'AE',
     require: ['^^ngcOmnibox', '^^ngcOmniboxSuggestions'],
-    scope: true,
+    scope: false,
     controller: NgcOmniboxSuggestionItemController,
     controllerAs: '$ctrl',
-    compile() {
+    compile(tElement) {
+      // Allows us to use document.getElementsByName which is fast AND returns a live-updating
+      // HTMLCollection
+      tElement.attr('name', 'ngc-omnibox-suggestion-item');
+      tElement.attr('role', 'option');
+
       return {
-        pre(scope, iElement, iAttrs, omnibox) {
+        pre(scope, iElement, iAttrs, [omnibox]) {
           scope.omnibox = omnibox;
+          scope.getIndex = () => omnibox.getSuggestionItemIndex(iElement[0]);
```

Comment:
Hmmm... does this still work without `false` scope? This seems quite dangerous to me.